src/askui/tools/askui/askui_controller.py

```diff
@@ -793,9 +789,6 @@ def get_process_list(
         Returns:
             controller_v1_pbs.Response_GetProcessList: Process list response containing:
                 - processes: List of ProcessInfo objects
-
-        Raises:

```

Comment:
What was the reason you removed those across all docstrings?

Reply:
See your point :) 

I lean more to the other side for the reason that a user could actually run into the error if the user did not connect beforehand (called the `connect()` method) and in this case the user just misused the lib and I would like to guide the user to fix that. I think `AssertionError` is just not a very descriptive name of that issue. Should be more something like `ConnectionError`.